iset.mm

```diff
@@ -144775,6 +144938,416 @@ be used in an extensible structure (slots must be positive integers).
+      qusaddvallemg $p |- ( ( ph /\ X e. V /\ Y e. V ) ->
+        ( [ X ] .~ .xb [ Y ] .~ ) = [ ( X .x. Y ) ] .~ ) $=
+        ( wcel w3a cfv co cec cqs cvv wer cbs basfn elex funfvex funfni sylancr
+        wfn syl eqeltrd erex quslem cv ercpbl imasaddvallemg 3ad2ant1 divsfvalg
+        sylc simp2 simp3 oveq12d wa 3ad2antl1 caovcld 3eqtr3d ) AKIUGZLIUGZUHZK
+        HUIZLHUIZEUJKLFUJZHUIKCUKZLCUKZEUJWDCUKAICULJEFHIUMKLNOPQABCDGHIUMMRSUD
+        AICUNZIUMUGZCUMUGTAIDUOUIZUMSADMUGZWIUMUGZUAWJUOUMVADUMUGWKUPDMUQWKUMDU
+        ODUOURUSUTVBVCZICUMVDVKUAVEABPVFQVFOVFZNVFZFCHIUMONTWLUDUCUBVGUEWLUFVHW
+        AWBWEWCWFEWABKCHIUMAVSWGVTTVIZAVSWHVTWLVIZUDAVSVTVLZVJWABLCHIUMWOWPUDAV
+        SVTVMZVJVNWABWDCHIUMWOWPUDWAONKLIIIFAVSWMIUGWNIUGVOWMWNFUJIUGVTUCVPWQWR
+        VQVJVR $.
+
+      $( The operation of a quotient structure is a function.  (Contributed by
+         Mario Carneiro, 24-Feb-2015.) $)
+      qusaddflemg $p |- ( ph ->
+        .xb : ( ( V /. .~ ) X. ( V /. .~ ) ) --> ( V /. .~ ) ) $=
+        ( cqs cvv wer wcel cbs cfv wfn basfn elexd funfvex sylancr eqeltrd erex
+        funfni sylc quslem cv ercpbl imasaddflemg ) AICUEJEFHIUFLMNOABCDGHIUFKP
+        QUBAICUGIUFUHCUFUHRAIDUIUJZUFQAUIUFUKDUFUHVDUFUHZULADKSUMVEUFDUIDUIUNUR
+        UOUPZICUFUQUSSUTABNVAOVAMVALVAFCHIUFMLRVFUBUATVBUCVFUDUAVC $.
+    $}
+
+    ${
+      qusaddf.p $e |- .x. = ( +g ` R ) $.
+      qusaddf.a $e |- .xb = ( +g ` U ) $.
+      $( The base set of an image structure.  (Contributed by Mario Carneiro,
+         24-Feb-2015.) $)
+      qusaddval $p |- ( ( ph /\ X e. V /\ Y e. V ) ->
+        ( [ X ] .~ .xb [ Y ] .~ ) = [ ( X .x. Y ) ] .~ ) $=
+        ( vx cv cec cmpt cvv eqid cqs wer wcel cbs cfv wfn basfn funfvex funfni
+        elexd sylancr erex sylc qusval quslem imasplusg cplusg plusgslid slotex
+        eqeltrd syl eqeltrid qusaddvallemg ) AUCBCDEFUCGUCUDBUEUFZGUGHIJKLMNOPQ
+        RSTVLUHZAGBUIEDCFVLGJKLAUCBCFVLGUGJOPVMAGBUJGUGUKBUGUKQAGCULUMZUGPAULUG
+        UNCUGUKVNUGUKZUOACJRURVOUGCULCULUPUQUSVHGBUGUTVAZRVBPAUCBCFVLGUGJOPVMVP
+        RVCRUAUBVDAECVEUMZUGUAACJUKVQUGUKRCVEJVFVGVIVJVK $.
+
+      $( The base set of an image structure.  (Contributed by Mario Carneiro,
+         24-Feb-2015.) $)
+      qusaddf $p |- ( ph ->
+        .xb : ( ( V /. .~ ) X. ( V /. .~ ) ) --> ( V /. .~ ) ) $=
+        ( vx cv cec cmpt cvv eqid cqs wer wcel cbs cfv wfn basfn funfvex funfni
+        elexd sylancr erex sylc qusval quslem imasplusg cplusg plusgslid slotex
+        eqeltrd syl eqeltrid qusaddflemg ) AUABCDEFUAGUAUBBUCUDZGUEHIJKLMNOPQRV
+        JUFZAGBUGEDCFVJGHIJAUABCFVJGUEHMNVKAGBUHGUEUIBUEUIOAGCUJUKZUENAUJUEULCU
+        EUIVLUEUIZUMACHPUPVMUECUJCUJUNUOUQVFGBUEURUSZPUTNAUABCFVJGUEHMNVKVNPVAP
+        STVBAECVCUKZUESACHUIVOUEUIPCVCHVDVEVGVHVI $.
+    $}
+
+    ${
+      qusmulf.p $e |- .x. = ( .r ` R ) $.
+      qusmulf.a $e |- .xb = ( .r ` U ) $.
+      $( The base set of an image structure.  (Contributed by Mario Carneiro,
+         24-Feb-2015.) $)
```

Comment:
"The base set of an image structure." -> "The multiplication in a quotient structure." (also in set.mm).